superset/views/sql_lab.py

```diff
@@ -75,6 +75,9 @@ def pre_update(self, obj):
 
 
 class SavedQueryViewApi(SavedQueryView):
+    list_columns = [
+        'label', 'sqlalchemy_uri', 'user_email', 'schema', 'description',
```

Comment:
Do we need `sqlalchemy_uri` and `user_email` in the list view, how about the database name and user name in instead?

Reply:
Oh actually I didn't realized this is `SavedQueryViewApi ` and not just `SavedQueryView`, it's totally good/fine to add it here. I thought this would affect the list view that users see...